client/src/main/kotlin/no/nav/su/se/bakover/client/stubs/oppdrag/SimuleringStub.kt

```diff
@@ -66,7 +62,7 @@ class SimuleringStub(
                     { null },
                     { it.gjeldendeForDato(måned.fraOgMed) },
                 )
-                val nyLinje = tidslinjeNyUtbetaling.gjeldendeForDato(måned.fraOgMed)
+                val nyLinje = tidslinjeEksisterendeOgNy.gjeldendeForDato(måned.fraOgMed)
```

Comment:
@jacob-meidell Man kan ikke lenger be om tidslinje for en utbetalingsserie som kun har Reaktivering, siden den ikke kan knyttes til en Ny linje. Vil dette fungere likt?